content/faqs/can-i-get-data-for-my-aoi.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Can I get data for my project area?
+title: Can I get data for my specific project area?
 tags: data
+category: getting-data
```

Comment:
thanks for consistently naming the categories using this shorthand. We can map it to the more human readable strings in the app.

Reply:
No problem.